app/assets/javascripts/discourse/app/components/modal/history.js

```diff
@@ -166,11 +167,18 @@ export default class History extends Component {
     });
   }
 
+  get firstPaint() {
+    return this.loading && this.initialPaint;
+  }
```

Comment:
Does it have to check `this.loading` too or could you just use `this.initialPaint` in the template?